packages/api/webpack.config.js

```diff
@@ -26,6 +30,19 @@ module.exports = {
     sfSemaphoreDown: './lambdas/sf-semaphore-down.js',
     sfStarter: './lambdas/sf-starter.js'
   },
+  devtool: process.env.PRODUCTION ? false : 'inline-source-map',
+  resolve: {
+    alias: {
+      'saml2-js': path.resolve(__dirname, 'node_modules/saml2-js/lib-js/saml2.js'),
+      'ejs': path.resolve(__dirname, 'node_modules/ejs/ejs.min.js')
```

Comment:
what is this?

Reply:
ejs is a dependency that was throwing an warning in webpack, This just aliases the requires to use the minified version and removes the warning.